Documentation/ceph-filesystem.md

```diff
@@ -128,6 +128,25 @@ Create the storage class.
 kubectl create -f cluster/examples/kubernetes/ceph/csi/cephfs/storageclass.yaml
 ```
 
+## Mirroring
+
+Since Ceph Pacific, CephFS supports asynchronous replication of snapshots to a remote CephFS file system via cephfs-mirror tool. Snapshots are synchronized by mirroring snapshot data followed by creating a snapshot with the same name (for a given directory on the remote file system) as the snapshot being synchronized.
+It is generally useful when planning for Disaster Recovery.
+For clusters that are geographically distributed and stretching is not possible due to high latencies.
+
```

Comment:
How do you setup the peer? Still working on that?

Reply:
Yes, just waiting for https://github.com/ceph/ceph/pull/39050